x-pack/plugin/src/test/resources/rest-api-spec/test/data_stream/10_data_stream_resolvability.yml

```diff
@@ -230,8 +243,8 @@
       indices.get_settings:
         index: ".ds-simple-data-stream1-*"
 
-  - is_false: \.ds-simple-data-stream1-000001.settings.index.lifecycle
-  - is_false: \.ds-simple-data-stream1-000002.settings.index.lifecycle
+  - is_false: ${idx0name}.settings.index.lifecycle
```

Comment:
I'm not sure why a test fails on this line when a very similar line below succeeds:

https://github.com/elastic/elasticsearch/pull/65205/files#diff-bb60a4ca6d613aa0f52c15e7278930763876d350d28af32297f2dac488316798R57